presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java

```diff
@@ -247,7 +268,10 @@ public void setup()
                 "real",
                 "double",
                 "decimal(10,5)",
-                "decimal(30,10)"
+                "decimal(30,10)",
+
+                "varchar_direct",
+                "varchar_dictionary"
```

Comment:
you mentioned that there it is not guaranteed that dictionary encoding will be used even if all values are the same; is this the case? @highker James, do you know why would that be the case?

Reply:
@mbasmanova  its with the varchar_direct, even we are using the ramdomAscii string generation it uses a dictionary reader sometimes